tools/pg_inspector/cli.rb

```diff
@@ -1,12 +1,14 @@
 require 'trollop'
 require 'pg_inspector/util'
 require 'pg_inspector/active_connections_to_yaml'
+require 'pg_inspector/servers_to_yaml'
 
 module PgInspector
   class Cli
     attr_accessor :cmd
     SUB_COMMANDS = {
-      :active_connections => ::PgInspector::ActiveConnectionsYAML
+      :active_connections => ::PgInspector::ActiveConnectionsYAML,
+      :servers            => ::PgInspector::ServersYAML
```

Comment:
You're already in the `PgInspector` namespace so you should be able to use the following above:

```
      :active_connections => ActiveConnectionsYAML,
      :servers            => ServersYAML
```